src/cli/commands/common.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+
+export type PluginName = "git" | "github";
```

Comment:
I don't think this file is in the right spot.
- the set of standard plugins isn't a CLI-specific concept, it is generic to the project as a whole
- even if it were CLI specific, I think the "commands" subdir loses its meaning if we put files there that don't correspond to commands. (note: historically, the reason I created a commands/ dir was to satisfy oclif's requirement that there be a 1:1 mapping between commands in a command dir and actual commands; however, since we have arbitrary path redirection, that isn't a constraint on our source tree. but, I think it's still nice to maintain that organization.)

Reply:
[offline discussion]: until we have a need for this elsewhere, I'm happy to have it under `cli/` (just not `cli/commands`)